src/Hosting/test/FunctionalTests/ShutdownTests.cs

```diff
@@ -28,6 +28,7 @@ public class ShutdownTests : LoggedTest
         [ConditionalFact]
         [OSSkipCondition(OperatingSystems.Windows)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
+        [Flaky("https://github.com/aspnet/AspNetCore-Internal/issues/2577", FlakyOn.AzP.Linux)]
```

Comment:
@Tratcher is already doing this in #10852

Reply:
Thanks, reverted the change in my PR in favor of https://github.com/aspnet/AspNetCore/pull/10852